skeleton/ci/gitops-template/azure/rhtap/env.sh

```diff
@@ -2,7 +2,7 @@
 export REBUILD=${REBUILD-true}
 export SKIP_CHECKS=${SKIP_CHECKS-true}
 
-CI_TYPE=${CI_TYPE:-jenkins}
+CI_TYPE=${CI_TYPE:-azure}
```

Comment:
This means going forward `CI_TYPE` would be mandatory variable?

Reply:
Ignore it. Got confused because of `Jenkins` value

Reply:
Oh wait, is the default value change intentional? I'd prefer to not change it, just in case this breaks other things.